packages/now-routing-utils/src/schemas.ts

```diff
@@ -2,94 +2,115 @@
  * An ajv schema for the routes array
  */
 export const routesSchema = {
-  type: 'array',
-  maxItems: 1024,
-  items: {
-    type: 'object',
-    additionalProperties: false,
-    properties: {
-      src: {
-        type: 'string',
-        maxLength: 4096,
-      },
-      dest: {
-        type: 'string',
-        maxLength: 4096,
-      },
-      methods: {
-        type: 'array',
-        maxItems: 10,
-        items: {
-          type: 'string',
-          maxLength: 32,
-        },
-      },
-      headers: {
-        type: 'object',
-        additionalProperties: false,
-        minProperties: 1,
-        maxProperties: 100,
-        patternProperties: {
-          '^.{1,256}$': {
-            type: 'string',
-            maxLength: 4096,
-          },
-        },
-      },
-      locale: {
-        type: 'object',
-        additionalProperties: false,
-        properties: {
-          redirect: {
-            type: 'object',
-            additionalProperties: false,
-            minProperties: 1,
-            maxProperties: 100,
-            patternProperties: {
-              '^.{1,256}$': {
-                type: 'string',
-                maxLength: 4096,
-              },
-            },
-          },
-          value: {
-            type: 'string',
-            maxLength: 4096,
-          },
-          path: {
-            type: 'string',
-            maxLength: 4096,
-          },
-          cookie: {
-            type: 'string',
-            maxLength: 4096,
-          },
-          default: {
-            type: 'string',
-            maxLength: 4096,
-          },
-        },
+	type: 'array',
+	maxItems: 1024,
+	items: {
+	  anyOf: [
+		{
+		  type: 'object',
+		  required: ['src'],
+		  additionalProperties: false,
+		  properties: {
+			src: {
+			  type: 'string',
+			  maxLength: 4096,
+			},
+			dest: {
+			  type: 'string',
+			  maxLength: 4096,
+			},
+			headers: {
+			  type: 'object',
+			  additionalProperties: false,
+			  minProperties: 1,
+			  maxProperties: 100,
+			  patternProperties: {
+				'^.{1,256}$': {
+				  type: 'string',
+				  maxLength: 4096,
+				},
+			  },
+			},
+			methods: {
+			  type: 'array',
+			  maxItems: 10,
+			  items: {
+				type: 'string',
+				maxLength: 32,
+			  },
       },
-      handle: {
-        type: 'string',
-        maxLength: 32,
+      important: {
+			  type: 'boolean',
       },
       continue: {
         type: 'boolean',
       },
-      check: {
-        type: 'boolean',
-      },
-      important: {
-        type: 'boolean',
-      },
-      status: {
-        type: 'integer',
-        minimum: 100,
-        maximum: 999,
-      },
-    },
-  },
+			check: {
+			  type: 'boolean',
+			},
+			status: {
+			  type: 'integer',
+			  minimum: 100,
+			  maximum: 999,
+			},
+			locale: {
+			  type: 'object',
+			  additionalProperties: false,
+			  required: ['redirect', 'cookie'],
```

Comment:
@ijjk Are the redirect/cookie props required on the locale?

Reply:
`cookie` is optional as far as I know but `redirect` is required for the `locale` route

Reply:
I've updated it so that all properties are optional but it requires at least one to be provided. That way it is matching all of the tests so everything should be fine.

I had to change the type assertions for errors though because the schema changed and the returned errors are not the same. The only "problem" is that error messages would be different maybe not that intuitive but that's a price to pay to have a strict validation. Before this updated the contents of some objects could include anything.